### PR TITLE
Adding Cisco CVE-2019-1606

### PR DESCRIPTION
**Pull request by santosomar**
Adding Cisco CVE-2019-1606

### CODE REVIEW

File: 2019/1xxx/CVE-2019-1606.json [hunk 1]
```diff
@@ -1,18 +1,109 @@
 {
```
Comment (cve-team) [inline 2019/1xxx/CVE-2019-1606.json]: Invalid control character at: line 32 column 63 (char 1358)

Comment (cve-team) [inline 2019/1xxx/CVE-2019-1606.json]: Expecting object: line 48 column 25 (char 2072)

Comment (santosomar) [reply 2019/1xxx/CVE-2019-1606.json]: Removing extra space that cause this exception.

Comment (santosomar) [reply 2019/1xxx/CVE-2019-1606.json]: Edited file accordingly.

**Pull request merged**